_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+drupal_planet_summary: |
+  How to move from serving your site over HTTP to HTTPS. Spoiler alert: there are some easy ways to do this with various Drupal hosting options. A quick read for guidance.
+---
+
+This is part two of a two-part series.
+
+In [part one](/2018/06/26/why-you-need-https.html), we cogently convinced you that regardless of what your organization does and what functionality your website has, it is in your best interest to serve your website securely over HTTPS exclusively. Here we provide some guidance as to how to make the move.
+
+## How to transition to HTTPS
+
+To fully transition to HTTPS from HTTP means to serve your website HTML and assets exclusively over HTTPS. This requires a few basic things:
+
+- A digital certificate from a certificate authority (CA)
+- Proper installation of the certificate on your website's server
+- Ensuring all assets served from your website are served over HTTPS
+
+Let's break those down
+
+### Acquire a digital certificate
+
+As briefly discussed in [part one](/2018/06/26/why-you-need-https.html), to implement HTTPS for your website you must procure a digital certificate from a certificate authority. Just like domain name registrars lease domain names, CAs lease digital certificates for a set time period. Each certificate has a public and private component. The public component is freely shared and allows browsers to recognize that a "trusted" CA was the one to distribute the certificate and it is also used to for encrypt data transmitted from the browser. The private complement is only shared with the purchaser of the certificate, and can uniquely decrypt data encrypted by the public key. CAs use various methods through email or DNS to "prove" that the person who purchased the certificate is a rightful administrator of the domain for which they purchased it. Once you've received the private key associated with the certificate, you'll need to install it on your server. Annual certificate costs range from up to $1,000 to free. More on that in a moment.
+
+### Install the certificate
+
+![Lots of wires]({{ site.url }}/assets/img/blog/install-https-analogy.jpg)
+
+Installing an HTTPS certificate manually on a server is a non-trivial engineering task. We [explain this at a high-level in part one](/2018/06/26/why-you-need-https.html#what-authentication-does-for-you). It  requires expertise from someone who is experienced and comfortable administering servers. There are many different installation methods unique to each permutation of server software and hosting platform, so I won't expend any real estate here attempting to lay it out. If you have to do a manual installation, it's best to search your hosting provider's documentation. However, depending on the complexity of your website architecture, there are ways to ease this process. Some hosting platforms have tools that substantially simplify the installation process. More on _that_ in a moment as well.
+
```

Comment:
Suggest - "Installing an HTTPS certificate manually on a server is ~a non-trivial~ not a trivial engineering task." I think this is easier to read.